samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/compose_tests/WebComposeTest.kt

```diff
@@ -39,22 +39,22 @@ class WebComposeTest : TestCase() {
                 }
             }
 
-            step("Find \"Sign in\" button and \"Protect your data\" title") {
+            step("Find \"Sign in\" button and title") {
```

Comment:
Maybe it makes sense to create our own simple WebPage?

Reply:
Yeah, It make sense. I'll work on that